test/e2e/config/operator.yaml

```
@@ -40,8 +40,8 @@ variables:
   ARTIFACTS_FOLDER: "_artifacts"
   HELM_BINARY_PATH: "helm"
   HELM_EXTRA_VALUES_FOLDER: "/tmp"
-  KUBERNETES_VERSION: "v1.31.0"
-  KUBERNETES_MANAGEMENT_VERSION: "v1.30.0"
+  KUBERNETES_VERSION: "v1.30.3"
+  KUBERNETES_MANAGEMENT_VERSION: "v1.30.3"
```

```suggestion
  KUBERNETES_VERSION: "v1.30.4"
  KUBERNETES_MANAGEMENT_VERSION: "v1.30.4"
```

This eliminates errors in crust-gather.